osu.Game/Overlays/Changelog/ChangelogBuild.cs

```diff
@@ -67,23 +68,35 @@ private void load(OsuColour colours)
 
                 foreach (APIChangelogEntry entry in categoryEntries)
                 {
-                    LinkFlowContainer title = new LinkFlowContainer
+                    var entryColour = entry.Major ? colours.YellowLight : Color4.White;
```

Comment:
Can this be assigned directly to the icon color, it's inconsistent with how the icon is assigned: `Icon = entry.Type == ...`